core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/PlatformType.java

```diff
@@ -74,7 +74,7 @@ public Run createRun() {
       return new IonTorrentRun();
     }
   }, //
-  PACBIO("PacBio", "8Pac", "SMRT Cell", "SMRT Cells", null) {
+  PACBIO("PacBio", "SMRT Cells", "SMRT Cell Contents", "SMRT Cell Contents", null) {
```

Comment:
`containerName` ("SMRT Cell") should be singular to match the others, as this is used in other places too. We do want the table name to be plural on the Edit Run page though. To do that, you can edit `editRun.jsp` and add an "s" to the name of `list_container`. Adding an "s" works for all platforms, so no need to add another property to the enum.